pages/15.Reports/docs.en.md

```diff
@@ -34,6 +34,16 @@ Some reports have graphs available. Select the graph desired from the left list
 
 ## Dashboard Widget
 
-Each graph of each report is made available as a widget on the dashboard allowing complete customization of the dashboard. 
- 
-![](widget.png)
\ No newline at end of file
+Each graph of each report is made available as a widget on the dashboard allowing complete customization of the dashboard.
+
+![](widget.png)
+
+## Schedule
+
+Enable or disable sending reports via emails by using the toggle switch. You can schedule emails to send reports to one or more email address. In the To field, enter a comma-separated list of email addresses and set the frequency of sending reports by choosing day, week, or month from the drop-down list. To be able to schedule emails to send reports, run the command:
```

Comment:
```suggestion
Enable or disable sending reports via email by using the toggle switch. 

You can schedule emails to send reports to one or more email addresses. In the To field, enter a comma-separated list of email addresses and set the frequency of sending reports by choosing day, week, or month from the drop-down list. 

To be able to send the scheduled reports, the following cron command is required - the argument allows you to specify a report by ID if required.
```

Reply:
Re-worked and moved the paras around. Please see if it looks fine now.